packages/cdktf-cli/bin/cdktf.ts

```diff
@@ -24,11 +24,6 @@ if (!CDKTF_DISABLE_PLUGIN_CACHE_ENV) {
   process.env.TF_PLUGIN_CACHE_DIR = ensurePluginCache();
 }
 
-if (semver.lt(process.version, "10.12.0")) {
-  console.error("Need at least Node v10.12 to run");
-  process.exit(1);
-}
```

Comment:
Is this being removed since inaccurate and doesn't really work? If so, would be nice to get it working some other way.

Reply:
We have the check environment function that takes care of it centrally with the right condition

Reply:
It's here: https://github.com/hashicorp/terraform-cdk/blob/fd88df429238ec85bf5d7550e58c19146f10cc65/packages/cdktf-cli/bin/cmds/helper/check-environment.ts#L46